lib/main.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:delivery_app/screens/categories_screen.dart';
+import 'package:flutter/material.dart';
+import 'package:delivery_app/data/data.dart';
+
+void main() {
+  runApp(const MyApp());
+}
+
+class MyApp extends StatelessWidget {
```

Comment:
Please change the name of the widget to something meaningful.
```suggestion
class DeliveryApp extends StatelessWidget {
```